roles/pulp/defaults/main.yaml

```diff
@@ -1,10 +1,18 @@
 ---
-pulp_image: quay.io/pulp/pulp:stable
-pulp_ports:
-  - "8080:80"
+pulp_api_image: quay.io/pulp/pulp-minimal:stable
+pulp_content_image: quay.io/pulp/pulp-minimal:stable
+pulp_worker_image: quay.io/pulp/pulp-minimal:stable
+
+pulp_api_ports:
+  - "24817:80"
+pulp_content_ports:
+  - "24816:80"
```

Comment:
I don't think we want both of these exposing their ports on "80" but rather just exposing their respective ports, i.e. `24816:24816".

Reply:
We switched to host networking now, so this won't be used anymore.
Makes me wonder tho, how will we scale the API: can't start more than one container with the same port (no idea if that's needed tho).

Reply:
Today we only have a single service that exposes `pulpcore-api`'s `--workers` parameter. Isn't that the same with a container?

Reply:
Technically? Sure. It just feels weird (in a container context, I'd scale by just adding more containers).
We have the same problem with Foreman/Puma.

But luckily we don't have to solve it here and now.

Reply:
In a quadlet context you can probably share a systemd activated socket and make it `pulp-api@` so you can scale it up but I think that's premature

Reply:
opened https://github.com/theforeman/foreman-quadlet/issues/51 to track this further